server/applications.json

```diff
@@ -648,8 +648,10 @@
                         "windows": [
                             "C:\\Program Files\\Side Effects Software\\Houdini 19.5.805\\bin\\houdini.exe"
                         ],
-                        "darwin": [],
-                        "linux": []
+                        "darwin": [
+                            "/Applications/Houdini/Houdini19.5.805/Frameworks/Houdini.framework/Versions/Current/Resources/bin/houdini"
+                        ],
+                        "linux": ["/opt/hfs19.5.805/bin/houdini"]
```

Comment:
I think it hasn't `/bin`. 
I don't use the app on linux but I saw this in a [YT tutorial](https://www.youtube.com/watch?v=ZPR5lp7GSkc). 
![image](https://github.com/user-attachments/assets/82186b42-29cc-4efd-b103-4347775e35ee)

```suggestion
                        "linux": ["/opt/hfs19.5.805/houdini"]
```

Reply:
Some client communication is showing that there is indeed a `/bin` subfolder.

[private thread](https://discord.com/channels/517362899170230292/1268146246917296232/1268892825529876634)

I think is this particular video it works from that folder because he runs `source houdini_setup` which initializes the houdini env and _likely_ adds the `bin/` folder to `PATH` or alike.

I'm quite confident the paths are correct as they are now :D

Reply:
jap this works because of the source Houdini setup. 
it allows you to run Houdini from where ever you want.